Cargo.toml

```diff
@@ -23,6 +23,7 @@ byteorder = "1.3"
 clap = "2.32"
 derive_more = "0.14"
 exonum = "0.11.0"
+exonum-merkledb = { git = "https://github.com/exonum/exonum.git", rev = "33e45eac5c9d710a100ce80b56d2d31a5b1f6dfc" }
```

Comment:
I guess there should be TODO here to remove `rev`.

Reply:
Can't you use reexported merkledb? https://github.com/exonum/exonum/blob/master/exonum/src/lib.rs#L56

Reply:
MerkleDB is not published yet.

Reply:
This one will be fixed after release.